tests/wpt/web-platform-tests/css/css

```diff
@@ -0,0 +1,8 @@
+<!doctype html>
```

Comment:
Maybe this file is added accidentally 👀?

Reply:
oops, sorry.

Reply:
nvm 😉